go.mod

```diff
@@ -139,7 +138,6 @@ require (
 )
 
 require (
-	github.com/deepmap/oapi-codegen v1.12.4
```

Comment:
wut

Reply:
Mr bot wants us to keep things clean I guess.

Reply:
I kinda don't wanna add a dependabot ignore on this. Uh, let's snooze this one time and resolve it for reals next time?

Reply:
I nuked the commits and did the upgrade manually. Tests were failing before. IDK what dependabot was doing here, total mess.